src-tauri/src/cpu_miner.rs

```diff
@@ -141,27 +147,36 @@ impl CpuMiner {
         // Refresh CPUs again.
         s.refresh_cpu_all();
 
-        // let cpu_brand = s.get_global_processor_info().unwrap().brand().to_string();
         let mut cpu_brand = "Unknown";
         for cpu in s.cpus() {
             cpu_brand = cpu.brand();
-            println!("{}", cpu.brand());
-            println!("{}", cpu.name());
+            break;
         }
```

Comment:
is this `for` needed anymore?

Reply:
it's messy, I can refactor it into a get(0) instead